packages/inngest/src/types.ts

```diff
@@ -991,6 +991,10 @@ export interface RegisterRequest {
   deployId?: string;
 }
 
+export interface IntrospectionCapabilities {
```

Comment:
```suggestion
export interface Capabilities {
```

Rename because capabilities are reported in both introspection and the sync request